src/platform-apps/channels/image-cards/image-model.ts

```diff
@@ -0,0 +1,6 @@
+export interface ImageModel {
```

Comment:
we can combine this file with src/platform-apps/channels/image-cards/image-sizes.ts, because it has only types, maybe we can name the file types.ts